metrics-operator/controllers/common/providers/prometheus/prometheus.go

```diff
@@ -149,3 +181,31 @@ func getResultForVector(result model.Value, r *KeptnPrometheusProvider) (string,
 	}
 	return value, b, nil
 }
+
+func getResultForStepMatrix(result model.Value, r *KeptnPrometheusProvider) ([]string, []byte, error) {
+	// check if we can cast the result to a matrix
+	resultMatrix, ok := result.(model.Matrix)
+	if !ok {
+		return nil, nil, fmt.Errorf("could not cast result")
+	}
+
+	if len(resultMatrix) == 0 {
+		r.Log.Info("No values in query result")
```

Comment:
I think we can remove the logging here, as any errors returned by this component will be logged later in the [controller](https://github.com/keptn/lifecycle-toolkit/blob/5b471203e412a919903876212ac45c04f180e482/metrics-operator/controllers/metrics/controller.go#L101). This also applies for the `getResultForVector` function

Reply:
Fixed